week-3/E-array-map/exercise.js

```diff
@@ -2,4 +2,7 @@
 // Write multiple solutions using different syntax (as shown in the README)
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
-
+var multipliedBy = numbers.map(number => {
+  return number * 100;
```

Comment:
Well done!
The instructions mention trying the same thing with different syntax - can you add those in as well please?

Reply:
sorry but not sure what do you mean what I need to do

Reply:
Look at the README.md file in the E-array-map folder - there is a "Callback functions" section in there - there are examples of writing the same function in different ways. Please add those to your exercise here.